src/Samples/Multiselect.Core/Program.fs

```diff
@@ -54,4 +54,4 @@ let main window =
 
   WpfProgram.mkSimple init update bindings
   |> WpfProgram.withLogger (new SerilogLoggerFactory(logger))
-  |> WpfProgram.startElmishLoop window
+  |> WpfProgram.startElmishLoop window
```

Comment:
FYI the empty line at the end of the files was added as a convenience for git diffs.

Reply:
Sure.  I used to add final blank line until I started using GitKraken.  Unlike every other git diff visualization I have seen (such as by Source Tree or GitHub), I think their diff visualization is better when there is no ending blank line.

I don't feel strongly about this though.